tests/test_0001_html.py

```diff
@@ -336,3 +344,46 @@ def test_html_strings():
     node = HTML('<button disabled="true">Click me</button>')[0]
 
     assert node.disabled
+
+    # data binding nodes ######################################################
+    node = HTML('<input value="abc"/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'abc'
+
+    node = HTML('<input type="text" value="xyz" disabled/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'xyz'
+    assert node.disabled is True
```

Comment:
If you test if inputs can be disabled, you should also test if the previous input ``disabled`` is False

Reply:
Agree. Fixed.